src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberAnnotationElement.java

```diff
@@ -1,5 +1,10 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.magicnumber;
 
+/* Config:
+ *
+ * ignoreAnnotationElementDefaults = false
+ */
+
 /**
  * ignoreAnnotationElementDefaults = false
```

Comment:
This Javadoc comment is redundant now. Please remove.

Reply:
Done